src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -1165,21 +1166,37 @@ BOOST_AUTO_TEST_CASE(SimpleSigmoidCrossEntropyLayerTest)
   double error2 = module.Forward(std::move(input2), std::move(target2));
   BOOST_REQUIRE_SMALL(error2 / input2.n_elem - 1.5027283, 1e-6);
 
+  input3 = arma::mat("0 -1 -1 0 -1 0 0 -1");
+  target3 = arma::mat("0 -1 -1 0 -1 0 0 -1");
+  double error3 = module.Forward(std::move(input3), std::move(target3));
+  BOOST_REQUIRE_SMALL(error3 / input3.n_elem - 0.00320443, 1e-6);
```

Comment:
This is just fine, just wanted to point out that you could also use: ``BOOST_REQUIRE_CLOSE``.